docs/Documentation/UserGuide/8-Tools-watermark.md

```diff
@@ -0,0 +1,202 @@
+<!--
```

Comment:
Hi,
Where should this doc be added? If this document should be added in "User Guide" (as "Memory Estimation Tool"), file "0-content.md" need to be updated at the same time. Or if it is added in "Tools" (as "Grafana / Spark Connector"), I will change the website end. : )

Reply:
"Tools" (as "Grafana / Spark Connector") I think.
This pr has not been reviewed and merged into master yet, so the website should not be changed now I guess.